src/ad/azext_ad/generated/_help.py

```diff
@@ -0,0 +1,148 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['ad ds'] = """
+    type: group
+    short-summary: Manage domain service with ad
+"""
+
+helps['ad ds list'] = """
+    type: command
+    short-summary: "The List Domain Services in Resource Group operation lists all the domain services available under \
```

Comment:
not short. It is very long...What about moving it to `long-summary`？

Reply:
Refined in `manual/_help.py`